frame/liberland-legislation/src/lib.rs

```diff
@@ -34,24 +34,24 @@ pub mod pallet {
 
     #[pallet::weight(10_000)]
     pub fn add_law(origin: OriginFor<T>, index1: u32, index2:u32, lawContent: BoundedVec<u8, ConstU32<65536>> ) -> DispatchResult {
-    	let sender = ensure_signed(origin)?;
+    	ensure_root(origin)?;
```

Comment:
Origin should reflect on what type of referendum type it's coming from(does it have external majority origin?) like here:
https://github.com/liberland/liberland_substrate/blob/main/frame/democracy/src/lib.rs#L815

Let's change this origin not to be root, rather reflect on what type of referendum the Origin comes from, we only want referendums that are executed to call this functions, not the root key user